ops/configuration/yii-conf/index.dev.php.dist

```diff
@@ -4,7 +4,7 @@
 # To be effective, ensure the two lines below are placed before we load Yii.php
 define('YII_DEBUG',$YII_DEBUG);
 define('YII_TRACE_LEVEL',$YII_TRACE_LEVEL);
-define('DISABLE_CACHE',$DISABLE_CACHE);
+define('DISABLE_CACHE',false);
```

Comment:
**suggestion:** Since variable `DISABLE_CACHE` has been hard-coded, so the variable `DISABLE_CACHE` declared in `.env` is redundant and can be removed? 
**suggestion:** Should the hardcode the `DISABLE_CACHE` in `index.staging.php.dist`  and `index.live.php.dist` to `false`, as caching should be always `true` for the production environments as suggested in this PR?

Reply:
Hi @kencho51,

Good thinking, but not the right thing to do.
On production environments, we need to be able to enable/disable any piece of infrastructure on demand. (e.g: in the case of caching, what if the cache is full, or some bugs not due to caching but made worse by caching happen, etc...)
Environment variable is the way to do that, given that we've adopted an immutable deployment approach (i.e: the codebase+configuration is immutable, so we cannot - and we don't want to do that anyways - connect to the machine and change manually the config). 

So on production, we still need to have this variable coming from the environment, and the functionality being responsive to its value.